### ISSUE TITLE
Missing `--cluster-name` parameter to cloud-controller-manager binary

### ISSUE THREAD

**Issue by titilambert** [kubernetes/kubernetes#45557]
**Kubernetes version** (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.2", GitCommit:"477efc3cbe6a7effca06bd1452fa356e2201e1ee", GitTreeState:"clean", BuildDate:"2017-04-19T20:33:11Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.2", GitCommit:"477efc3cbe6a7effca06bd1452fa356e2201e1ee", GitTreeState:"clean", BuildDate:"2017-04-19T20:22:08Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Environment**:
- **Cloud provider or hardware configuration**: Azure
- **OS** (e.g. from /etc/os-release): Red Hat 7.3
- **Kernel** (e.g. `uname -a`): Linux envdev01-kubenode-xs-1 3.10.0-514.16.1.el7.x86_64 1 SMP Fri Mar 10 13:12:32 EST 2017 x86_64 x86_64 x86_64 GNU/Linux
- **Install tools**: from binaries

**What happened**: cloud-controller-manager doesn't support `--cluster-name`. So it creates a Azure load balancer named `kubernetes` instead of cluster name.

**What you expected to happen**:
`cloud-controller-manager` should support --cluster-name then yhe Azure load balancer name should be the cluster name

**How to reproduce it** (as minimally and precisely as possible):
```
cloud-controller-manager  --cluster-name
unknown flag: --cluster-name
...
```

**Comment by luxas** [comment]
@wlan0 @thockin @roberthbailey @colemickens

**Comment by jhorwit2** [comment]
This was fixed by https://github.com/kubernetes/kubernetes/pull/52524. Should we cherry-pick this  @luxas?

**Comment by luxas** [comment]
I think the current state is fine with that it's included in v1.8
